### ISSUE TITLE
Profile does not have an `updatesOnAccessTokenChange` property.

### ISSUE THREAD

**Issue by BrentMifsud** [facebook/facebook-ios-sdk#1766]
## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-ios-sdk/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](https://github.com/facebook/facebook-ios-sdk/blob/master/CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Xcode Version: `12.6`
- Swift Version: `5.4` (if issue is Swift related)
- Installation Platform & Verison: `SPM`

## Goals

enable profile updates on access token change

## Expected Results

profile updates on token change enabled

## Actual Results

Profile does not have member: `updatesOnAccessTokenChange`.

Is the documentation [here](https://developers.facebook.com/docs/swift/reference/structs/userprofile.html) incorrect? 

## Steps to Reproduce

What are the steps necessary to reproduce this issue?

## Code Samples & Details

Please provide a code sample, as well as any additional details, to help us track down the issue. If you can provide a link to a test project that allows us to reproduce the issue, this helps us immensely in both the speed and quality of the fix.

**Note:** Remember to format your code for readability:

```swift
import FacebookCore
import FacebookLogin
import GoogleSignIn

extension AppDelegate {
	func startFacebook(application: UIApplication, launchOptions: [UIApplication.LaunchOptionsKey: Any]?) {
		ApplicationDelegate.shared.application(application, didFinishLaunchingWithOptions: launchOptions)
		Profile.updatesOnAccessTokenChange = true // compile error here
	}
}
```

**Comment by BrentMifsud** [comment]
My Xcode code complete wasnt working at the time I was trying to implement this.

It looks like the documentation IS incorrect.

working usage is:

```swift
Profile.enableUpdatesOnAccessTokenChange(true)
```

**Comment by joesus** [comment]
> Is the documentation here incorrect?

Yes. Correct documentation is at: https://developers.facebook.com/docs/reference/iossdk/current/FBSDKCoreKit/classes/fbsdkprofile.html
